docs/content/v0.20/user-guide.md

```diff
@@ -0,0 +1,31 @@
+# User Guide
+
+Status of this document: it is the barest of a start. Much more needs to be written.
+
+This document is for users of a release. Examples of using the latest stable release are in [the examples document](examples.md). This document adds information to support other usage scenarios.
+
+## Using an existing Kind cluster as the hosting cluster
+
+The examples say to create a Kind cluster for hosting using the following command.
+
+```shell
+kflex init --create-kind
+```
+
+To use a pre-existing Kind cluster instead, make sure that your current kubeconfig context is for accessing that cluster and issue the following command.
```

Comment:
The pre-existing kind cluster has 2 requirements
1. TLS passthrough
2. additional port 9443

Reply:
Revised in the force-push to 78693606a

Reply:
The force-push to a6d3e703a adds one clarification.

Reply:
note that you could use a port different from 9443. Different port can be specified by the -p option in `kflex init`
See 
```
Usage:
  kflex init [flags]

Flags:
  -s, --chatty-status       chatty status indicator (default true)
  -c, --create-kind         Create and configure a kind cluster for installing Kubeflex
  -d, --domain string       domain for FQDN (default "localtest.me")
  -p, --externalPort int    external port used by ingress (default 9443)
  -h, --help                help for init
  -k, --kubeconfig string   path to kubeconfig file
  -v, --verbosity int       log level
 ```

Reply:
anyway, this could be too much detail at this level